docs/FAQ.md

```diff
@@ -198,11 +214,48 @@ A phase of a `machine` can be identified with `Machine.Status.CurrentStatus.Phas
 
 * `Pending`: Machine creation call has succeeded. MCM is waiting for machine to join the cluster.
 * `CrashLoopBackOff`: Machine creation call has failed. MCM will retry the operation after a minor delay.
-* `Running`: Machine creation call has succeeded. Machine has joined the cluster successfully.
-* `Unknown`: Machine health checks are failing, eg `kubelet` has stopped posting the status.
-* `Failed`: Machine health checks have failed for a prolonged time. Hence it is declared failed. `MachineSet` controller will replace such machines immediately.
+* `Running`: Machine creation call has succeeded. Machine has joined the cluster successfully and corresponding node doesn't have `node.gardener.cloud/critical-components-not-ready` taint.
+* `Unknown`: Machine [health checks](#what-health-checks-are-performed-on-a-machine) are failing, eg `kubelet` has stopped posting the status.
+
+* `Failed`: Machine health checks have failed for a prolonged time. Hence it is declared failed by `Machine` controller. `Failed` machines get replaced, but in a rate limited fashion of 1 machine per machinedeployment.
```

Comment:
`Unknown` machines get replaced in a rate-limited fashion and not `Failed` ones.

Reply:
done